lib/trino-parquet/src/main/java/io/trino/parquet/writer/ColumnWriter.java

```diff
@@ -22,7 +22,11 @@
 
 public interface ColumnWriter
 {
-    void writeBlock(ColumnChunk columnChunk)
+    /**
+     * @param columnChunk Values of a column for a parquet row-group
+     * @return Count of currently buffered bytes
+     */
+    long writeBlock(ColumnChunk columnChunk)
```

Comment:
Fair point. I've now updated this to avoid change in API and make `PrimitiveColumnWriter` track changes to bufferedBytes internally rather than re-computing it on every `getBufferedBytes` call.